### ISSUE TITLE
Subtype list not appearing properly when editing character

### ISSUE THREAD

**Issue by SpeedyD** [corowne/lorekeeper#224]
### Bug as reported:

(Main branch, last commit parent: [f447da64](https://github.com/corowne/lorekeeper/commit/f447da64b28dd5a2a9ceb5329defaa7876270739))

**URL of page:** `<siteurl>/character/<slug>`
**Description of bug:** Attempting to edit the subtype of a character makes the subtype not show up.
**How was the bug caused (exact steps to reproduce):** Go to any character page, click the Edit button on the Info tab, discover that the Subtype list still says 'Pick a Species First' even though it's an existing character with a species set already.
### Further details

Fairly minor bug, was gonna attempt to fix it myself but got too busy unfortunately.

TL;DR the Subtype list doesn't fill up when you edit a character, even if it has a Species picked already. _Seems_ fairly straightforward, just needs to trigger the refresh once the Modal opens, instead of waiting until someone swaps the Species.

**Comment by SpeedyD** [comment]
Feature added in PR #229, part of the develop branch. (v3.0.0 milestone.)
